doc/tutorials/gapi/table_of_content_gapi.markdown

```diff
@@ -15,3 +15,16 @@ how G-API module can be used for that.
     is ported on G-API, covering the basic intuition behind this
     transition process, and examining benefits which a graph model
     brings there.
+
+- @subpage tutorial_gapi_face_beautification
+
+    *Languages:* C++
+
+    *Compatibility:* \> OpenCV 4.1.2
+
+    *Author:* Orest Chura
+
+    This is a tutorial where a computer vision algorithm (that consists of a networks
```

Comment:
I think `consists of an inference of several networks and ...` sounds better

Reply:
Can accept it